apps/admin-x-activitypub/src/views/Profile/components/ProfilePage.tsx

```
@@ -0,0 +1,181 @@
+                                        onUnfollow={noop}
+                                    />
+                                    }
+                                </div>
+                                <Heading className='mt-4' level={3}>{!isLoadingAccount ? account.name : <Skeleton className='w-32' />}</Heading>
+                                <a className='group/handle mt-1 flex items-center gap-1 text-[1.5rem] text-gray-800 hover:text-gray-900' href={account.url} rel='noopener noreferrer' target='_blank'>
+                                    <span>{!isLoadingAccount ? account?.handle : <Skeleton className='w-full max-w-56' />}</span>
+                                    <Icon className='opacity-0 transition-opacity group-hover/handle:opacity-100' name='arrow-top-right' size='xs'/>
+                                </a>
+                                {(account.bio || customFields.length > 0) && (<div ref={contentRef} className={`ap-profile-content transition-max-height relative text-[1.5rem] duration-300 ease-in-out [&>p]:mb-3 ${isExpanded ? 'max-h-none pb-7' : 'max-h-[160px] overflow-hidden'} relative`}>
+                                    {!isLoadingAccount ?
+                                        <div dangerouslySetInnerHTML={{__html: account?.bio ?? ''}} /> :
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)